app/html/settings/friend-hops.js

```diff
@@ -0,0 +1,84 @@
+const nest = require('depnest')
+const { h, computed, onceTrue, Value, watch } = require('mutant')
+const merge = require('lodash/merge')
+
+exports.gives = nest({
+  'app.html.settings': true
+})
+
+exports.needs = nest({
+  'app.html.settings': 'map',
+  'config.sync.get': 'first',
+  'config.sync.getCustom': 'first',
+  'config.sync.setCustom': 'first'
+})
+
+const friendsHops = 2
+exports.create = function (api) {
+  return nest({
+    'app.html.settings': friendHopsConnections
+  })
+
+  function friendHopsConnections () {
+    let hops = Value(api.config.sync.get('friends.hops', friendsHops))
+    hops.initial = hops()
+
+    watch(hops, hops => {
+      const intHops = parseInt(hops)
+      updateConfig(intHops)
+    })
+
+    return {
+      title: 'Friend Hops',
+      body: h('Friends', [
+        h('div.description', [
+          'Specify whose content you replicate, and thus the size and shape of your network.'
```

Comment:
Probably we need some helper test explaining what 0, 1, 2, .. means (1 = just people you followed, 2 = people you follow and their friends). That or a link out to [6 degress of kevin bacon](https://en.wikipedia.org/wiki/Six_Degrees_of_Kevin_Bacon)